galaxy.yml

```diff
@@ -1,12 +1,13 @@
 ---
 namespace: middleware_automation
 name: amq_streams
-version: "0.0.7"
+version: "0.0.8"
```

Comment:
@rpelisse The galaxy version is not updated manually, right? I mean, it is implemented by the release pipeline with a new tag is created, so we don't need to update the `galaxy.yml` file? Just to double check before merging this PR.

Reply:
@rmarting if you want, I can rollback that change